Makefile

```diff
@@ -244,7 +244,7 @@ simphony-plugins: simphony-kratos simphony-numerrin simphony-mayavi simphony-ope
 	@echo
 	@echo "Simphony plugins installed"
 
-simphony-framework:
+simphony-framework: simphony simphony-plugins
```

Comment:
just thought it is a bit misleading to print "simphony framework installed" when it actually does nothing